coalib/parsing/ConfParser.py

```diff
@@ -15,7 +15,8 @@ def __init__(self,
                  key_delimiters=(',', ' '),
                  section_name_surroundings=None,
                  remove_empty_iter_elements=True):
-        section_name_surroundings = section_name_surroundings or {"[": "]"}
+        
+	section_name_surroundings = {"[": "]"} if section_name_surroundings is None else section_name_surroundings
```

Comment:
The space removed before line has happened in a lot of places  - please check them all out (not commenting everywhere)